routes/game.rb

```diff
@@ -139,6 +139,15 @@ class Api
           # POST '/api/game/<game_id>/kick
           r.is 'kick' do
             game.remove_player(r.params['id'])
+
+            game.to_h
+          end
+
+          halt(400, 'You are not allowed to change the player order in this game') unless game.settings['player_order']
```

Comment:
remove this halt

Reply:
Then I'll also remove the line from the game card and the option from create_game, because this way any game can have the order manually changed.